docs/install/installEverest.md

```diff
@@ -1,4 +1,4 @@
-# Install Everest
+# Install Percona Everest using CLI
```

Comment:
Can we include a note about the fact that starting 1.4.0, the CLI uses the Helm chart to install Everest. Something like this:

```
Starting from version 1.4.0, `everestctl` uses the Helm chart for installing Everest. To configure chart parameters during installation via the CLI, you can:

* Use the `--helm-set` flag to specify individual parameter values.
* Provide a values file with the `--helm-values` flag for bulk configuration.
```

Reply:
Added.